src/modules/map/components/cesium/CesiumMap.vue

```diff
@@ -376,7 +376,7 @@ export default {
             globe.showGroundAtmosphere = false
             globe.showWaterEffect = false
             // increases the LOD (Cesium will load one tile further down the zoom pyramid) => higher rez WMTS
-            globe.maximumScreenSpaceError = 0.5
+            //globe.maximumScreenSpaceError = 0.5
```

Comment:
Better remove this as simply comment, or if it is just temporary please comment the reason of the temporary removal